setup.py

```diff
@@ -20,7 +20,7 @@
     install_requires=[
```

Comment:
The code looks mostly fine, but there are a few changes I would recommend:

1. **Versioning:** If this is the initial commit for the project, the version number "0.0.1" is fine. However, if this is an update, you should increment the version number.

2. **Package URL:** The URL "https://github.com/EricRahm/atsy" should be updated if the package is hosted elsewhere.

3. **Dependencies:** The version of "psutil" is fixed to "5.6.6". It's generally better to specify a minimum version unless there's a specific reason to require that exact version.

4. **Entry Points:** The entry point 'comp_analysis' points to 'example.comp_analysis:main'. If 'example' is not a module in your package, this will cause an error.

Here's the revised setup:

```markdown
setup(
    name="atsy",
    version="0.1.0",  # Update version if this is not the initial commit
    description="AreTheySlimYet",
    long_description="A set of tools for measuring cross-browser, cross-platform memory usage.",
    url="https://github.com/[Your_GitHub_Username]/atsy",  # Update URL if package is hosted elsewhere
    author="Eric Rahm",
    author_email="erahm@mozilla.com",
    license="MPL 2.0",
    classifiers=[
      "License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)"
    ],
    packages=["atsy"],
    install_requires=[
      "selenium",
      "marionette-harness",
      "psutil>=5.6.6",  # Specify minimum version
    ],
    entry_points={
      'console_scripts': ['comp_analysis=atsy.comp_analysis:main']  # Ensure 'atsy' is a module in your package
    }
)
```